src/components/HomePage/Banner.js

```diff
@@ -90,11 +117,19 @@ function renderInfo(classes) {
 					width: 150,
 					margin: 20
 				}} />
-			<Typography variant='h5' className={classes.text} component='h3'>
-				<time dateTime={hothStart.toISOString()}>
-					Feb 23<sup>rd</sup>, 2021
-				</time>
-			</Typography>
+			<Box className={classes.horizontalBox}>
```

Comment:
I think we can get rid of the `horizontalBox` class and specify the styles here.
```suggestion
			<Box display='flex' alignItems='center' color='white'>
```